components/Snackbar/src/MDCSnackbarManager.h

```diff
@@ -238,7 +245,7 @@
  Default is set to NO.
  */
 @property(nonatomic, readwrite, setter=mdc_setAdjustsFontForContentSizeCategory:)
-    BOOL mdc_adjustsFontForContentSizeCategory;
+    BOOL mdc_adjustsFontForContentSizeCategory NS_DEPRECATED_IOS(8_0, 11_0);
```

Comment:
Please add a deprecation message that explains the expected alternative API for iOS 11+.

For example, 

```objc
- (nullable UIViewController *)challengeComposeControllerWithPlayers:(nullable NSArray<NSString *> *)playerIDs message:(nullable NSString *)message completionHandler:(__nullable GKChallengeComposeCompletionBlock)completionHandler NS_DEPRECATED_IOS(7_0, 8_0, "pass GKPlayers to challengeComposeControllerWithMessage:players:") ;
```